wagtailio/static/sass/components/_cta.scss

```diff
@@ -42,13 +46,16 @@
         > svg {
             fill: $color--white;
             transform: translateX(10px) scale(1.1);
+
+            @media (forced-colors: active) {
+                fill: currentColor;
+            }
         }
     }
 
     &__title {
         color: var(--color--interaction);
         margin-bottom: 10px;
-        transition: color $transition;
```

Comment:
not sure why this has been removed

Reply:
This was removed as part of another minor issue mentioned in the audit (not whcm related). The CTA button currently has inconsistent transition speeds between the title and subtitle in light mode.

Reply:
Ok, in this case, can you please add this issue to the PR description